sdk/provisioning/Azure.Provisioning.Communication/tests/BasicCommunicationTests.cs

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Resources;
+using Azure.Provisioning.Tests;
+
+namespace Azure.Provisioning.Communication.Tests;
+
+public class BasicCommunicationTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    public async Task CreateCommunicationServices()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(
+            ctx =>
+            {
+                BicepParameter location = BicepParameter.Create<string>(nameof(location), BicepFunction.GetResourceGroup().Location);
+                location.Description = "Service location.";
+
+                CommunicationService comm =
+                    new(nameof(comm), "2023-03-31")
+                    {
+                        Location = new StringLiteral("global"),
+                        DataLocation = "unitedstates"
+                    };
```

Comment:
And on that same point, to anyone playing with early bits: nothing is broken, you just haven't expressed yourself hard enough.   There are tons of little things that either aren't picked up correctly by the generator or are poorly spec'ed or who knows... but no worries - you can do whatever with an expression.  In this example, `Location` generates as the well-known [AzureLocation](https://learn.microsoft.com/en-us/dotnet/api/azure.core.azurelocation?view=azure-dotnet#properties) but it expects a magic `"global"` instead.  But never fear - you can always duct tape more expressions over stuff.